recoder/src/main/java/recoder/java/JavaSourceElement.java

```diff
@@ -192,4 +192,8 @@ public final ProgramFactory getFactory() {
     public String toSource() {
         return factory.toSource(this);
     }
+
```

Comment:
Until now, we only very sparsely added to the recoder repo. Since it is part of our sources now and nobody maintains it: I'd say we can optimise things here, too. 
@unp1 do you concur?

Reply:
Discussed in current meeting: It is part of our code base and is open for any modifications.